src/app/common/components/DataTable/DataTable.tsx

```diff
@@ -64,40 +79,85 @@ const DataTable = (props: DataTableProps) => {
         </StyledRow>
       );
     }
+    if (operatorListFlow && isXsWindowSize) {
+      return data.map((row: any[], rowIndex: number) => {
+        return (
+          <Grid key={`row-key-${rowIndex}`} xs={10} className={classes.TableStyledRow}>
+            {row.map((cell: any, cellIndex: number) => {
+              return (
+                <Grid key={`cell-key-${cellIndex}`} xs={cellIndex > 0 ? 4 : 12}>
+                  <StyledCell
+                    key={`cell-${cellIndex}`}
+                    align={headersPositions?.length ? headersPositions[cellIndex] : undefined}
+                    >
+                    <Typography className={classes.TableCellLabel}>{OPERATOR_CELL_LABEL_NAME[cellIndex]}</Typography>
+                    {cell}
+                  </StyledCell> 
+                </Grid>
+              );
+            })}
+          </Grid>
+      );
+      });
+    }
+    if (validatorListFlow && isXsWindowSize) {
```

Comment:
DataTable is common component and it should not know any specifics about operators and validators. It's better to move this logic out to operators/validators pages to outer scope, or bring here more common logic for displaying it differently

Reply:
done